File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    for size in sizes:
        if size[0]>size[1]:
            size[0],size[1] = size[1],size[0]
    max1,max2=0,0
    for size in sizes:
        if max1<size[0]:
            max1=size[0]
        if max2<size[1]:
            max2=size[1]
    return max1*max2

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    let x=0
    let y=0
    for(let size of sizes){
        if(size[0] < size[1]){
            let temp = size[0]
            size[0] = size[1]
            size[1] = temp
        }
        x = Math.max(x,size[0])
        y = Math.max(y,size[1])
    }
    return x*y
}